src/sentry/event_manager.py

```diff
@@ -156,12 +163,11 @@
 )
 PERFORMANCE_ISSUE_QUOTA = Quota(3600, 60, 5)
 
-DEFAULT_GROUPHASH_IGNORE_LIMIT = 3
 GROUPHASH_IGNORE_LIMIT_MAP = {
-    GroupType.PERFORMANCE_N_PLUS_ONE_DB_QUERIES: 3,
-    GroupType.PERFORMANCE_SLOW_DB_QUERY: 100,
-    GroupType.PERFORMANCE_CONSECUTIVE_DB_QUERIES: 15,
-    GroupType.PERFORMANCE_UNCOMPRESSED_ASSETS: 100,
+    PerformanceNPlusOneGroupType: PerformanceNPlusOneGroupType.ignore_limit,
+    PerformanceSlowDBQueryGroupType: PerformanceSlowDBQueryGroupType.ignore_limit,
+    PerformanceConsecutiveDBQueriesGroupType: PerformanceConsecutiveDBQueriesGroupType.ignore_limit,
+    PerformanceUncompressedAssetsGroupType: PerformanceUncompressedAssetsGroupType.ignore_limit,
 }
```

Comment:
Is this map used anymore?

Reply:
Oops no, it kept changing during rebases and I forgot to remove it.